#! /bin/sh /usr/share/dpatch/dpatch-run
## 13_always_display_warnings_and_failures.dpatch by Michael Prokop <mika@grml.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Always display failure and warning messages.

@DPATCH@
Index: live-boot-grml/scripts/boot/9990-initramfs-tools.sh
===================================================================
--- live-boot-grml.orig/scripts/boot/9990-initramfs-tools.sh	2012-11-28 08:10:50.815614786 +0100
+++ live-boot-grml/scripts/boot/9990-initramfs-tools.sh	2012-11-28 08:25:12.864563943 +0100
@@ -2,6 +2,17 @@
 
 #set -e
 
+# we definitely want this stuff visible
+log_failure_msg()
+{
+        printf "Failure: $@\n"
+}
+
+log_warning_msg()
+{
+        printf "Warning: $@\n"
+}
+
 log_wait_msg ()
 {
 	# Print a message and wait for enter
